### ISSUE TITLE
HTML link in the Readme.md does not work

### ISSUE THREAD

**Issue by Ravenwater** [posit-standard/posit-standard#2]
The link in the Readme.md that needs to link to HTML does not produce HTML;

It points to:
https://github.com/posit-standard/posit-standard/blob/master/_build/html/index.html

Should it not point to the readthedocs.org?

**Comment by agshew** [comment]
Yes, I've been meaning to update that. The link currently points to where it would be if it were built locally, but we should point to readthedocs.org

And we should probably have readthedocs.org point back to the GitHub project too.

**Comment by Ravenwater** [comment]
Andrew: agreed.

Also, I did review the PR, and thank you for including me in the change
review: I didn't know that github had that feature.

           *Dr. E. Theodore L. Omtzigt*

CEO and Founder

Stillwater Supercomputing, Inc.



East Coast Office : (617) 314 6424

West Coast Office: (415) 738 7387

Mobile phone #  : (916) 296-7901

3941 Park Drive, Suite 20-354

El Dorado Hills, CA 95762




On Mon, Jun 11, 2018 at 11:28 PM, Andrew Shewmaker <notifications@github.com
